### ISSUE TITLE
[FEATURE] windows hello

### ISSUE THREAD

**Issue by Sebleoy** [mylinuxforwork/dotfiles#191]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by mylinuxforwork** [comment]
@Sebleoy That would be a great feature. I agree.

**Comment by andresparrab** [comment]
Yeah that would be nice. Actually after having read this post I did install howdy and works for sddm but I have to click login first to start de camera login.. Is there anyway you achieve this so it will automatically start the camera and try to log in? Also how do you configure hyprlock to unlock with face recognition?
